package.json

```diff
@@ -23,11 +23,13 @@
     "test:ci": "npm run test -- --ci",
     "cypress": "cypress open",
     "cypress:headless": "cypress run",
+    "cypress:percy": "percy exec -- cypress run",
```

Comment:
We shouldn't need to run Percy locally, but I added some commands just in case. Still, you need to prepend the env var with the API key to be able to run it locally.